csrc/ops/alias.h

```diff
@@ -97,12 +97,14 @@ TORCH_CUDA_CU_API TensorView* transpose(TensorView* x);
 TORCH_CUDA_CU_API TensorView* pad(
     TensorView* x,
     const std::vector<Val*>& pad_widths,
-    Val* value = nullptr);
+    Val* value = nullptr,
+    std::optional<IterType> iter_type_opt = std::nullopt);
 
 //! Concatenate tensors in the given dimension
 TORCH_CUDA_CU_API TensorView* cat(
     const std::vector<TensorView*>& inputs,
-    int64_t dim);
+    int64_t dim,
+    std::optional<IterType> iter_type_opt = std::nullopt);
```

Comment:
These options let us pipe through an optional IterType to avoid symbolic IterTypes popping up during the RemoveEmptyPass. This is a bug that was exposed by the dynamic empty cat tests.